src/Command/DataMigration/MigrateRegionsToTableCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Command\DataMigration;
+
+use App\Entity\Centre;
+use App\Entity\Region;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'app:migrate-regions',
+    description: 'Migrate regions from string to object, this command is meant to be run only once',
+)]
+class MigrateRegionsToTableCommand extends Command
+{
+    public function __construct(private readonly EntityManagerInterface $em, string $name = null)
+    {
+        parent::__construct($name);
+    }
+
+    public const REGIONS_WITH_MAIL = [
+        'Auvergne-Rhône-Alpes' => 'aura@reconnect.fr',
+        'Bourgogne-Franche-Comté' => null,
+        'Bretagne' => 'bretagne@reconnect.fr',
+        'Centre-Val de Loire' => null,
+        'Corse' => null,
+        'Grand Est' => 'grand-est@reconnect.fr',
+        'Hauts-de-France' => 'hauts-de-france@reconnect.fr',
+        'Ile-de-France' => 'ile-de-france@reconnect.fr',
+        'Normandie' => 'normandie@reconnect.fr',
+        'Nouvelle-Aquitaine' => 'nouvelle-aquitaine@reconnect.fr',
+        'Occitanie' => 'occitanie@reconnect.fr',
+        'Pays de la Loire' => 'pays-de-la-loire@reconnect.fr',
+        'Provence-Alpes-Côte d’Azur' => 'paca@reconnect.fr',
+        'Autre' => null,
+    ];
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$this->promptConfirm($input, $output, $io)) {
+            return Command::SUCCESS;
+        }
+        $this->createRegions();
+        $this->migrateRegions($output);
+
+        $io->success('Done !');
+
+        return Command::SUCCESS;
+    }
+
+    private function createRegions(): void
+    {
+        foreach (self::REGIONS_WITH_MAIL as $regionName => $email) {
+            $region = (new Region())->setName($regionName)->setEmail($email);
+            $this->em->persist($region);
+        }
+        $this->em->flush();
+    }
+
+    private function migrateRegions(OutputInterface $output): void
+    {
+        $regionRepo = $this->em->getRepository(Region::class);
+        $centresRepo = $this->em->getRepository(Centre::class);
+        $centres = $centresRepo->findAllWithRegionAsString();
```

Comment:
Not that important, but you don't need intermediate variable

Reply:
$centres ? I actually use it to set the progress bar